samples/boards/stm32/mco/boards/nucleo_u5a5zj_q.overlay

```diff
@@ -0,0 +1,12 @@
+/* The clock that is output must be enabled. */
+&clk_lse {
+    status = "okay";
+};
+
+/ {
+    zephyr,user {
+        /* Select MCO pin to use. */
+        pinctrl-0 = <&rcc_mco_pa8>;
+        pinctrl-names = "default";
```

Comment:
Nit: is the property `pinctrl-names` really required in the `/zephyr,user` context?

Reply:
Yes, pinctrl-names is required in order for the pinctrl struct to be created. Removing it results in compilation error.